assets/js/script.js

```diff
@@ -119,209 +102,214 @@ var randomQuestion;
-    body.classList.toggle("dark");
+contrastToggle.addEventListener('click', function () {
+  body.classList.toggle('dark');
 });
 
 // for background animation
 function ready(document) {
-    document.querySelector(".wrapper").append("<ul class='circles'><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li></ul>");
-};
+  document
+    .querySelector('.wrapper')
+    .append(
+      "<ul class='circles'><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li></ul>"
+    );
+}
 
 function startGame() {
-    pageChangeOne();
-    score = 0;
-    scoreBoard.textContent = score;
-    timerCount = 60;
-    randomQuestion = questions.sort(() => Math.random() - .5);
-    currentQuestion = 0;
-    startTimer();
-    setNextQuestion();
+  pageChangeOne();
+  score = 0;
+  scoreBoard.textContent = score;
+  timerCount = 60;
+  randomQuestion = questions.sort(() => Math.random() - 0.5);
+  currentQuestion = 0;
+  startTimer();
+  setNextQuestion();
 }
 
 function pageChangeOne() {
-    container.forEach(function(containers) {
-        if(containers.classList.contains("one")) {
-            containers.classList.add("hide");
-        }
-        if(containers.classList.contains("two")) {
-            containers.classList.remove("hide");
-        }
-    });
-};
+  container.forEach(function (containers) {
+    if (containers.classList.contains('one')) {
+      containers.classList.add('hide');
+    }
+    if (containers.classList.contains('two')) {
+      containers.classList.remove('hide');
+    }
+  });
+}
 
-function pageChangeTwo(){
-    container.forEach(function(containers) {
-        if(containers.classList.contains("two")) {
-            containers.classList.add("hide");
-        };
-        if(containers.classList.contains("three")) {
-            containers.classList.remove("hide");
-        };
-    });
-    gameEndMessage();
-};
+function pageChangeTwo() {
+  container.forEach(function (containers) {
+    if (containers.classList.contains('two')) {
+      containers.classList.add('hide');
+    }
+    if (containers.classList.contains('three')) {
+      containers.classList.remove('hide');
+    }
+  });
+  gameEndMessage();
+}
 
 function startTimer() {
-    timer = setInterval(function() {
-        timerCount--;
-        secondsLeft.textContent = timerCount;
-        if (timerCount === 0 || questionCount > questions.length) {
-            clearInterval(timer);
-            pageChangeTwo();
-        };
-        if (timerCount <= 10) {
-            secondsLeft.style.color = "#ff8f8f";
-        };
-    },1000);
-};
+  timer = setInterval(function () {
+    timerCount--;
+    secondsLeft.textContent = timerCount;
+    if (timerCount === 0 || questionCount > questions.length) {
+      clearInterval(timer);
+      pageChangeTwo();
+    }
+    if (timerCount <= 10) {
+      secondsLeft.style.color = '#ff8f8f';
+    }
+  }, 1000);
+}
 
 function showQuestion(question) {
```

Comment:
Could this function be made to dynamically handle a question with one answer? 10 answers? Right now if there aren't exactly 4 answers, there's likely going to be an issue right?